google-cloud-dataproc/synth.metadata

```diff
@@ -45,5 +45,265 @@
         "config": "google/cloud/dataproc/artman_dataproc_v1beta2.yaml"
       }
     }
+  ],
+  "newFiles": [
```

Comment:
@dazuma I've never noticed this `newFiles` attribute in `synth.metadata` before, does it look OK to you?

Reply:
This is an attempt to detect when previously generated files are no longer generated (and it removes them). It's probably okay for now, but I'm not 100% sure about the implementation, so please keep an eye on any files that are deleted by future autosynth PRs and make sure they look legit.